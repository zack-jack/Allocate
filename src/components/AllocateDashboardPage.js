import React from "react";

import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";

const AllocateDashboardPage = () => (
  <div>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default AllocateDashboardPage;
