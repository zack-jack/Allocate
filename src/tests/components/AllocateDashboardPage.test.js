import React from "react";
import { shallow } from "enzyme";

import AllocateDashboardPage from "../../components/AllocateDashboardPage";

test("should render AllocateDashboardPage correctly", () => {
  const wrapper = shallow(<AllocateDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
