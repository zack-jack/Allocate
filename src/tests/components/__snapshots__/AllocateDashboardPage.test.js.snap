// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AllocateDashboardPage correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
